configs/default_cpu_air_params.json

```diff
@@ -0,0 +1,27 @@
+{
+    "field": "PrimeField0",
+    "channel_hash": "poseidon3",
+    "commitment_hash": "keccak256_masked160_lsb",
+    "n_verifier_friendly_commitment_layers": 9999,
+    "pow_hash": "keccak256",
+    "statement": {
+        "page_hash": "pedersen"
+    },
+    "stark": {
+        "fri": {
+            "fri_step_list": [
+                0,
+                4,
+                4,
+                3
+            ],
+            "last_layer_degree_bound": 128,
+            "n_queries": 10,
+            "proof_of_work_bits": 30
+        },
+        "log_n_cosets": 2
+    },
+    "use_extension_field": false,
+    "verifier_friendly_channel_updates": true,
+    "verifier_friendly_commitment_hash": "poseidon3"
+}
```

Comment:
Is this file still used? 

Maybe I'm out of the loop, but I thought we moved to using Moonsongs logic for generating these on the fly?

Reply:
Actually, the `fri_step_list` is the only part that should generated on the fly for now.
We use this file and `default_cpu_air_prover_config.json` to set the default values.

(fyi, it's no longer used in the Phase 2 implementation as the default values are set via CLI args)